pod/ai_enhancement/forms.py

```diff
@@ -0,0 +1,192 @@
+            attrs={
+                "aria-describedby": "id_titleHelp",
+            },
+        ),
+        help_text=_('''
+            Please choose a title between 1 and 250 characters.
+        '''),
+    )
+
+    description = forms.CharField(
+        label=_("Description"),
+        widget=forms.Textarea(
+            attrs={
+                "aria-describedby": "id_descriptionHelp",
+            },
+        ),
+        required=False,
+        help_text=_("Please choose a description."),
+    )
+
+    tags = TagField(
+        help_text=_('''
+            Please choose tags for your video.
+            Separate tags with spaces, enclose the tags consist of several words in quotation marks.
+        '''),
+        verbose_name=_("Tags"),
+    )
+
+    disciplines = forms.ModelChoiceField(
+        label=_("Discipline"),
+        queryset=Discipline.objects.all(),
+        required=False,
+        help_text=_('Please choose the discipline of your video.'),
+    )
+
+    fieldsets = [
+        (
+            "choose_title",
+            {
+                "legend": f"<i class='bi bi-info-lg' aria-hidden='true'></i>&nbsp;\
+                    {strings['title']['choose_information_string']}<br>\
+                    <div>\
+                        <div class='row'>\
+                            <div id='initial-version-title' class='col'>\
+                                <div\
+                                    class='border-d rounded-4 p-3 mb-3 mt-3 blockquote'\
+                                    title='{strings['title']['initial_information_t']}'>\
+                                        {strings['title']['initial_information']}\
+                                </div>\
+                            </div>\
+                            <div id='ai-version-title' class='col'>\
+                                <div\
+                                    class='border-d rounded-4 p-3 mb-3 mt-3 blockquote'\
+                                    title={strings['title']['ai_information_t']}>\
+                                        {strings['title']['ai_information']}\
+                                </div>\
+                            </div>\
+                        </div>\
+                    </div>",
+                "fields": ["title"],
+            },
+        ),
+        (
+            "choose_description",
+            {
+                "legend": f"<i class='bi bi-info-lg' aria-hidden='true'></i>&nbsp;\
+                {strings['description']['choose_information_string']}<br>\
+                <div>\
+                    <div class='row'>\
+                        <div id='initial-version-description' class='col'>\
+                            <div\
+                                class='border-d rounded-4 p-3 mb-3 mt-3 blockquote'\
+                                title='{strings['description']['initial_information_t']}'>\
+                                    {strings['description']['initial_information']}\
```

Comment:
ne pas mettre de title s'il répète exactement le texte affiché

Reply:
Le title est "Your initial description" tandis que le texte affiché est, en premier lieu,  "The initial description is loading…", puis la description de la vidéo. C'est pareil pour le `"ai_information"` et le `"ai_information_t"`.

Reply:
ah je n'avais pas vu le "initial_information_t" le petit "t" fait toute la différence, autant pour moi ;)